docs/production-deployment/cloud/nexus/getting-started.mdx

```diff
@@ -15,9 +15,11 @@ keywords:
 
 ---
 
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
 Temporal Nexus Public Preview is available within your Temporal Cloud account, with support in the [Temporal Go SDK](/develop/go).
 
-Calls across existing Namespaces can be enabled by creating a Nexus Endpoint in the Nexus API Registry, creating a Nexus Service in a Worker in the handler Namespace, and then using the Nexus Service from a caller Workflow in a different Namespace.
+Calls across existing Namespaces can be enabled by creating a Nexus Endpoint in the Nexus Registry, creating a Nexus Service in a Worker in the handler Namespace, and then using the Nexus Service from a caller Workflow in a different Namespace.
 
 Monolithic Namespaces can be decomposed into multiple Namespaces, by hiding service implementations behind a Nexus Endpoint in the monolithic Namespace, pointing all consumers at the new Nexus Endpoint, and then changing the Endpoint’s target Namespace to a different Namespace.
```

Comment:
Again, there is a _lot_ here for a brain to process at once. Consider breaking it down to make it easier for readers to digest. Also, you might want to motivate the advantages in maintenance, serviceability, responsiveness, etc in modular architectures vs monoliths. This line is so central to why Nexus rocks and it may deserve a little more love and a few extra sentences to get that across. Temporal is providing a well-designed pathway away from monoliths that developed from all kinds of circumstances to a better, shinier solution...through Nexus.

Reply:
fixed